src/writer/profile.ts

```diff
@@ -25,4 +29,6 @@ export async function action(message, ipfs): Promise<void> {
   };
 
   await db.queryAsync('REPLACE INTO users SET ?', params);
+
+  if (profile.avatar !== existingProfile.avatar) clearStampCache('avatar', message.from);
```

Comment:
Maybe better to wait until we finish the fetch, else user may see old avatar for some time,

Reply:
How could we achieve that ? What do you mean by finish the fetch ?

This is the method used by the UI (directly trigger this request after profile update), and I did notice that sometimes, the cache is not clearing correctly.

Reply:
I mean to wait until the cache is cleared.